### ISSUE TITLE
1.0 new features

### ISSUE THREAD

**Issue by egoist** [egoist/eme#48]
What we already know:

- Installer for Windows
- [Plugin system](https://github.com/egoist/eme/issues/39) - for custom themes and plugins
- [Presentation mode](https://github.com/egoist/eme/issues/17) - maybe as a plugin
- More export targets - Word (might be tough)/Image/EPUB

And something I'm not sure (maybe a plugin):

**Live editing mode**, create a shareable link and send it to someone else, then he/she can edit the same file you are editing, or you can set it read only.

**Comment by Perkovec** [comment]
More encodings

**Comment by Perkovec** [comment]
Plugins API is almost done, but need a separate class and refactoring

**Comment by egoist** [comment]
@Perkovec ok, it could be a new branch

**Comment by Perkovec** [comment]
@egoist for live editing mode needs back-end?

**Comment by egoist** [comment]
Yes, but that's not managed by the app, the app simply uses web socket to communicate with api server.

**Comment by Perkovec** [comment]
@egoist what's wrong with the Windows installer? I can create package

**Comment by egoist** [comment]
@Perkovec  There's no installer for windows now, only prebuilt binary.

**Comment by RoyiAvital** [comment]
Questions:

- Does it support Math using MathJax?
- Is it portable on Windows (No registry keys, using only local folder for settings, etc...)?

**Comment by egoist** [comment]
@RoyiAvital

- it already supports TeX
- yes
